tutorials/beginner/installing.md

```diff
@@ -1,27 +1,55 @@
+
+
+## Scan For New Things
+
+Now the functionality offered by the newly installed binding can be added. For the Network Info Binding this happens by starting an auto-discovery scan and will result in one new thing. A [thing](http://docs.openhab.org/concepts/things.html) is the logical representation of a physical entity, in this case your openHAB server.
+
+| 4. | ![](images/Installing_Extension_Discovery_Scan.png) | 1. In the left hand menu select the item **Inbox** <BR> 2. Click on the **Scan** icon in the upper right corner <BR> |
+| 5. | ![](images/Installing_Extension_Discovery_Execute.png) | In the now opening pop-up scroll down and click on  ![](images/Installing_Extension_Discovery_Start.png) in order to start the scan process for the **System Info Binding** |
+| 6. | ![](images/Installing_Extension_Discovery_Finished.png) | 1. The scan process will announce any newly found things via a small message box on the lower right hand side. ![](images/Installing_Extension_Discovery_Finished_Message.png) As there is only one server to discover, you can directly click the message box to jump to the inbox. <BR> 2. Alternatively you can click on **OK**. |
+| 7. | ![](images/Installing_Extension_Discovery_Result.png) |  As a result you will now find one new thing in the inbox. Now click on the add symbol ![](images/Installing_Hook.png). |
+| 8. | ![](images/Installing_Extension_Discovery_Add_Thing.png) | In the pop-up: <BR> 1. Leave the default name "Local computer" <BR> 2. click "ADD AS THING"  |
+| 9. | ![](images/Installing_Extension_Discovery_Add_Result.png) | Now the system info thing is added to your installation and has therefore moved out of the inbox. |
+
+## Configure Things
+
+| 10. | ![](images/Installing_Extension_Thing_Select.png) | 1. Click in the left hand menu to the item **Configuration** and then **Things**. You will see a list of the existing things. <BR> 2. Look for the item "Local computer" and press the ![](images/Installing_Extension_Thing_Select_Edit.png) edit symbol.   |
+| 11. | ![](images/Installing_Extension_Thing_Edit.png) | 1. Enter the computer's name on which the openHAB instance is running (here: Pollux) <BR> 2. Enter the location (here: Basement) <BR> 3. Click the ![](images/Installing_Hook.png) Save button. |
+| 12. | ![](images/Installing_Extension_Thing_Result.png) | As a result the overview of the thing you just created is shown. Please ensure that the status is ![](images/Installing_Extension_Thing_Result_Status.png). |
+| 13. | ![](images/Installing_Extension_Control.png) | To see the different information provided by the PaperUI, <BR> 1. navigate to the Controll page of the PaperUI. <BR> 2. There you will see a tab (2.) with the location you entered in stetp 13 (here: Basement). On this tab the thing representing your openHAB server displays various information from the System Info binding. |
```

Comment:
The tables are there on purpose in order to give more structure to the document. And with my local Jekyll installation the output looks fairly ok. However the size of the images could be better aligned. Do you know how this can be done?

![Uploading Screenshot.png…]()